cmake/README.md

```diff
@@ -19,6 +19,7 @@ TBB_INSTALL_VARS:BOOL - Enable auto-generated vars installation(packages generat
 TBB_VALGRIND_MEMCHECK:BOOL - Enable scan for memory leaks using Valgrind (OFF by default)
 TBB_DISABLE_HWLOC_AUTOMATIC_SEARCH - Disable HWLOC automatic search by pkg-config tool (OFF by default)
 TBB_ENABLE_IPO - Enable Interprocedural Optimization (IPO) during the compilation (ON by default)
+TBB_BUILD_APPLE_FRAMEWORKS - Enable building Apple .frameworks instead of dylibs, only available on Apple platform. (OFF by default)
```

Comment:
```suggestion
TBB_BUILD_APPLE_FRAMEWORKS - Enable the Apple* frameworks instead of dylibs, only available on the Apple platform. (OFF by default)
```

Reply:
@aepanchi I'm not sure why Apple needs a asterisk. Is it to mark it as copyright? If so, shouldn't the 2nd Apple in this string also have an asterisk?
Tbh, I'm also not 100% about the 'the' in front of Apple in this sentence. But then again, I'm not a native English speaker :).

Reply:
@geertbleyen Yes, it is for the copyright purposes. Just one for the first mention in the file is enough

Reply:
Ok, fixed.